docs/UserGuide.md

```diff
@@ -27,7 +27,7 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
    * `list` : Lists all contacts.
 
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the StaffConnect contacts.
```

Comment:
I noticed that the portions below only say `to the / in the contacts` whereas for this line here, it says `to the StaffConnect contacts`. Is there any particular reason for this?

Reply:
I don't really understand what the  question is referring to here, Staffconnect contacts refers to the contacts list in the StaffConnect

Reply:
I have reworded the statement to ensure clarity now!